lib/ansible/modules/cloud/amazon/ec2_elb_facts.py

```diff
@@ -80,6 +80,14 @@
 
 import traceback
 
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import (
```

Comment:
I'd suggest it's an undocumented preference

But it's a fairly standard way of doing things, at least in the amazon modules:

```
$ git grep  'import [^(]*,' | wc -l
75
$ git grep 'import (' | wc -l
5
```